po/it.po

```diff
@@ -463,21 +463,21 @@ msgstr ""
 #: ../virtaal/tips.py:31
 msgid "To mark the current translation as fuzzy, simply press <Alt+U>."
 msgstr ""
-"Per segnare la presente traduzione come \"fuzzy\", basta premere <Alt+U>."
+"Per segnare la presente traduzione come fuzzy, basta premere <Alt+U>."
```

Comment:
The translator is highlighting that the word fuzzy here is untranslated or borrowed, so leave the quotes.  The quotes checker is a guide not a hard and fast rule.  This we'd consider a false positive.